### DISCUSSION TITLE
请问这个报错怎么解决？

### DISCUSSION THREAD

**Discussion by Darkerami** [nonebot/discussions#302] [问答]
![微信图片_20230127163408](https://user-images.githubusercontent.com/118607168/215042864-654a5490-e02b-44d3-968a-6e53c410a8e3.png)
请问这个报错怎么解决？

**Comment by yanyongyu** [comment]
你这上报地址都没配好

**Reply by Darkerami** [reply]
这个解决了谢谢求看下下一条只能接收消息不能回复咋办呀
